angular/pom.xml

```diff
@@ -23,6 +23,7 @@
     <artifactId>zeppelin</artifactId>
     <groupId>org.apache.zeppelin</groupId>
     <version>0.6.0-incubating-SNAPSHOT</version>
+    <relativePath>..</relativePath>
```

Comment:
While parent's pom.xml is in parent directory, i think it's safe to omit `<relativePath>..</relativePath>`. Is there any special reason explicitly defining it?